components/language-chooser/common/find-language/package.json

```diff
@@ -7,10 +7,11 @@
   "main": "./index.js",
   "types": "./index.d.ts",
   "scripts": {
-    "langtag-processing": "ts-node-esm ./langtagProcessing.ts",
+    "langtag-processing": "tsx ./langtagProcessing.ts",
```

Comment:
ts-node doesn't work well with node >= 20 :( 

https://github.com/TypeStrong/ts-node/issues/1997
https://github.com/paranext/paranext-core/commit/7bb5219a0#diff-840902f027c26280334dfa9c48e1e11c2c1caabd5adb91eb000533dd8b35c235R5

Reply:
Even if we're going sticking with node 18 for now, I figure we might as well keep this change so that we can upgrade node in the future
<!-- Reviewable comment -OBSJvau3zxus9tALMTw:b-pu1kv3 -->
<!-- Sent from Reviewable.io -->